_contentTemplates/common/get-started.md

```diff
@@ -226,13 +217,19 @@ To setup a local NuGet package source, so you can install the Telerik components
 
 
 #download-intro-para-for-get-started
-If you are familiar with the Telerik NuGet Feed and Blazor in general, you may want to follow the shorter, more technical article with the same information: [What You Need]({%slug getting-started/what-you-need%}). The current article is designed as a step-by-step tutorial from the basics for new users.
+If you are familiar with the Telerik NuGet Feed and Blazor in general, you may want to follow the shorter, more technical getting started article: [What You Need]({%slug getting-started/what-you-need%}). The current article is designed as a step-by-step tutorial for new users and starts from the basics.
 
 ## Step 0 - Download the Components
 
-A pre-requisite is having access to the Telerik UI for Blazor components. The easiest way to get them to your development machine is to use the <a href="https://www.telerik.com/download-trial-file/v2/control-panel" target="_blank">Progress Control Panel</a> or to download the [automated installer]({%slug installation/msi%}) from <a href="https://www.telerik.com/account/product-download?product=BLAZOR" target="_blank">your telerik.com account</a>.
```

Comment:
The tutorials should also focus on the trial users as an important group. That's why I suggest adding two sub-sections under "Step 0 - Download the Components" - the first section speaks to trialists and the second section speaks to developers with a license.